pkg/asset/ignition/machine/master_test.go

```diff
@@ -30,11 +31,9 @@ func TestMasterGenerate(t *testing.T) {
 					Region: "us-east",
 				},
 			},
-			Machines: []types.MachinePool{
-				{
-					Name:     "master",
-					Replicas: func(x int64) *int64 { return &x }(3),
-				},
+			ControlPlane: &types.MachinePool{
+				Name:     "master",
```

Comment:
is there a desire to eliminate the term "master" generally?

Reply:
just asking as that is a broader change to push through.

Reply:
Yes. The broader change of replacing the terms master and worker with control-plane and compute are tackled in #1227.